Tests/GraphQL/HyperwalletTransferMethodConfigurationTests.swift

```diff
@@ -64,15 +64,37 @@ class HyperwalletTransferMethodConfigurationTests: XCTestCase {
 
         bankAccountProcessingTime = graphQlResponse?.transferMethodTypes(countryCode: "CA", currencyCode: "CAD")?
             .first(where: { $0.code == "BANK_ACCOUNT" })
-            .flatMap { $0.processingTime }
-        XCTAssertNotNil(bankAccountProcessingTime)
-        XCTAssertEqual(bankAccountProcessingTime, "1 - 3 business days")
+            .flatMap { $0.processingTimes?.nodes?.first }
+        XCTAssertNotNil(bankAccountProcessingTime, "The procebankAccountProcessingTimessingTime should not be nil")
```

Comment:
Typo in procebankAccountProcessingTimessingTime

Reply:
Fixed